src/Algebra/Graph.hs

```diff
@@ -47,6 +47,7 @@ module Algebra.Graph (
   ) where
 
 import Control.Applicative (Alternative, (<|>))
+import Control.Deepseq (NFData (..))
```

Comment:
Looks like this should be `DeepSeq` instead -- see AppVeyor build log:

https://travis-ci.org/snowleopard/alga/jobs/300622428